template2.java

```diff
@@ -0,0 +1,48 @@
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.android.AndroidDriver;
+import io.appium.java_client.ios.IOSDriver;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testobject.appium.junit.TestObjectTestResultWatcher;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+public class AppiumTest {
+    @Rule
+    public TestObjectTestResultWatcher resultWatcher = new TestObjectTestResultWatcher();
+    private AppiumDriver driver;
+
+
+    @Before
+    public void setUp() throws MalformedURLException {
+        DesiredCapabilities caps = new DesiredCapabilities();
+
+$capabilities
+
+
+        driver = new IOSDriver(new URL("$domain"), caps);
```

Comment:
Does it matter it is an IOSDriver if we say enter an Android test? Shouldn't matter but want to double check.

Reply:
It does matter. I didn't tackle that yet, because we need to take into account all the cases (static allocation, dynamic allocation, etc.). I'm not sure what is the best way to have something that works right away without too much additional code.

Reply:
Cool just wanted to know. We could get fancy at some point and use Jinja or something like that to make more complex templates.

Reply:
Found a way to detect the OS via the API. This should know work for both OS.